contrib/python/pypodman/pypodman/lib/actions/pod/create_parser.py

```diff
@@ -0,0 +1,79 @@
+"""Remote client command for creating pod."""
+import sys
+
+import podman
+from pypodman.lib import AbstractActionBase, BooleanAction
+
+
+class CreatePod(AbstractActionBase):
+    """Implement Create Pod command."""
+
+    @classmethod
+    def subparser(cls, parent):
+        """Add Pod Create command to parent parser."""
+        parser = parent.add_parser('create', help='create pod')
+        super().subparser(parser)
+
+        parser.add_argument(
+            '--cgroup-parent',
+            dest='cgroupparent',
+            type=str,
+            help='Path to cgroups under which the'
+            ' cgroup for the pod will be created.')
+        parser.add_argument(
+            '--infra',
+            action=BooleanAction,
+            default=True,
+            help='Create an infra container and associate it with the pod'
+            '(default: %(default)s)')
+        parser.add_argument(
+            '-l',
+            '--label',
+            dest='labels',
+            action='append',
+            type=str,
+            help='Add metadata to a pod (e.g., --label=com.example.key=value)')
+        parser.add_argument(
+            '-n',
+            '--name',
+            dest='ident',
+            type=str,
+            help='Assign name to the pod')
```

Comment:
fence sitting.  Maybe "a name" or "the name", but just "name" might be fine too.